frontend/src/containers/AdminPanel/AdminPanel.js

```diff
@@ -36,6 +39,10 @@ class AdminPanel extends Component {
         users: this.createFullName(data),
       });
     });
+
+    setTimeout(() => {
+      this.setState({ render: true });
+    }, 2000);
```

Comment:
If we already have the user object, I'd just set `render` to true automatically. I'd only wait if you don't have the user.